src/component/dashboard/dashboard.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import * as categoryActions from '../../action/category';
+import CategoryForm from '../category-form/category-form';
+import CategoryItem from '../category-item/category-item';
+
+class Dashboard extends React.Component {
+  render() {
+    const { categories, categoryCreate } = this.props;
+    return (
+      <div className = 'dashboard'>
+        <CategoryForm onComplete = { categoryCreate }/> {
```

Comment:
`CategoryItem` is actually at the same level as `CategoryForm` in this component. So the way you've nested it is a little misleading for someone immediately scanning your JSX, I'd recommend doing something that makes it clear these components are siblings rather than being nested, i.e.:

```
  <CategoryForm onComplete = { categoryCreate }/> 
        {
        categories.map((currentCategory, i) => 
        <CategoryItem category = { currentCategory } key = { i }/>)
        }
```